luthier/config/src/config.lua

```diff
@@ -36,16 +37,34 @@ local metricConfig = metricsUtils.build_metric_config(metrics)
 local metricMakerConfig = metricMakerUtils.build_metric_maker_config(metricMakers)
 local physicalTableConfig, logicalTableConfig = tableUtils.build_table_config(tables)
 
+--- write to testResource/tableName
+local function write_table(targetDir, tableName, configTable)
+    parser.save(targetDir .. tableName, configTable)
+    print(targetDir .. " table " .. tableName .. " Updated now.")
+end
+
+local function config(tableName, configTable)
```

Comment:
I would probably call this `write_config` instead of just `config`. I typically only use
a noun as a function name in Lua when that function is going to be returning something.

Reply:
sure.